src/chapter_3/milestone_project_tic-tac-toe.md

```diff
@@ -105,6 +105,39 @@ impl Default for GameState {
 }
 ```
 
+> #### Warning
+>
+> Be careful when relying on Anchor to infer `space` using `T::default()`.
```

Comment:
We should really remove the default stuff in anchor and just require one to use `space`.